test/router.test.ts

```diff
@@ -97,6 +97,29 @@ describe("router", () => {
     const res = await request.get("/404");
     expect(res.status).toEqual(404);
   });
+
+  it("Handle shadowed route", async () => {
+    router.post(
+      "/test/123",
+      eventHandler((event) => `[${event.method}] ${event.path}`)
+    );
+
+    router.use(
+      "/test/**",
```

Comment:
Should we test with "**" as its closer to the Nuxt behaviour ?

Reply:
I wanted to also make sure base URL works. Notice that "both" have test in prefix. If base works, without certainly works too :)